01/sonar_sweep.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
I'd suggest using 'black' over your code as standard to get consistent looking code.